PYTHON parameter for python executable, and use python2 by default, by
vadmium@aur
--- ipkg-utils-050831/Makefile
+++ ipkg-utils-050831/Makefile
@@ -1,4 +1,5 @@
 UTILS = ipkg-build ipkg-upload
+PYTHON=python2
 DESTDIR=
 
 PREFIX=/usr/local
@@ -6,7 +7,7 @@ PREFIX=/usr/local
 all: build
 
 build: ipkg-compare-versions
-	python setup.py build
+	$(PYTHON) setup.py build
 
 ipkg-compare-versions: ipkg-compare-versions.c
 	$(CC) $(CFLAGS) -o ipkg-compare-versions ipkg-compare-versions.c
@@ -14,7 +15,7 @@ ipkg-compare-versions: ipkg-compare-vers
 install: ${UTILS}
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
 	cp ${UTILS} $(DESTDIR)$(PREFIX)/bin
-	python setup.py install --root=$(DESTDIR)
+	$(PYTHON) setup.py install --root=$(DESTDIR)
 	chmod agu+rx ipkg-make-index
 	cp -f ipkg-make-index $(DESTDIR)$(PREFIX)/bin
